src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -14,17 +14,22 @@
 use Sulu\Bundle\WebsiteBundle\Resolver\ParameterResolverInterface;
 use Sulu\Component\Webspace\Analyzer\RequestAnalyzerInterface;
 use Symfony\Bundle\TwigBundle\Controller\ExceptionController as BaseExceptionController;
+use Symfony\Component\Debug\Exception\FlattenException;
```

Comment:
Well, if we want only to be compatible with 2.8 and 3.x only one of these exceptions would be necessary. The other `FlattenException` is already deprecated and the new one already exists in 2.8. It's only a problem if somebody is using something which has already been deprecated in Symfony 2.3.

Apart from that I can't imagine that anybody using Sulu creates its own `FlattenException`, but let the framework create these exceptions for them. So I don't think it would be a serious BC break to just use the new one. /cc @chirimoya